Tools/WebKitTestRunner/ios/UIScriptControllerIOS.mm

```diff
@@ -605,11 +606,13 @@ static void setModifierFlagsForUIPhysicalKeyboardEvent(UIPhysicalKeyboardEvent *
         keyboardEvent._modifierFlags = modifierFlags;
 }
 
-static UIPhysicalKeyboardEvent *createUIPhysicalKeyboardEvent(NSString *hidInputString, NSString *uiEventInputString, UIKeyModifierFlags modifierFlags, UIKeyboardInputFlags inputFlags, bool isKeyDown)
+enum class IsKeyDown : bool { No, Yes };
+
+static UIPhysicalKeyboardEvent *createUIPhysicalKeyboardEvent(NSString *hidInputString, NSString *uiEventInputString, UIKeyModifierFlags modifierFlags, UIKeyboardInputFlags inputFlags, IsKeyDown isKeyDown)
 {
     auto* keyboardEvent = [getUIPhysicalKeyboardEventClass() _eventWithInput:uiEventInputString inputFlags:inputFlags];
     setModifierFlagsForUIPhysicalKeyboardEvent(keyboardEvent, modifierFlags);
-    auto hidEvent = createHIDKeyEvent(hidInputString, keyboardEvent.timestamp, isKeyDown);
+    auto hidEvent = createHIDKeyEvent(hidInputString, keyboardEvent.timestamp, isKeyDown == IsKeyDown::Yes ? true : false);
```

Comment:
Nit - this should also just be `isKeyDown == IsKeyDown::Yes`.

Reply:
Fixed